packages/snaps-utils/src/manifest/manifest.ts

```diff
@@ -191,6 +192,20 @@ export async function checkManifest(
     );
   }
 
+  if (!snapFiles.svgIcon) {
+    warnings.push(
+      'No icon found in `source.location.npm.iconPath`. It is highly recommended for your Snap to have an icon.',
```

Comment:
What do you think of something like this?

> No icon found in the Snap manifest. It is recommended to include an icon for the Snap. See https://link.to.docs for more information.

Reply:
Any preferred link for this?